recipes/r-gam/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
```

Comment:
bat is used for windows builds, but bioconda is not building for the Win platform.